github/orgs_users_blocking.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// ListBlockedUsers lists all the users blocked by an organization.
+//
+// GitHub API docs: https://developer.github.com/v3/orgs/blocking/#list-blocked-users
+func (s *OrganizationsService) ListBlockedUsers(ctx context.Context, org string, opt *ListOptions) ([]*User, *Response, error) {
+	// TODO: add a check for scope to be admin:org
```

Comment:
You have this `// TODO: add a check for scope to be admin:org` in 8 places. You also asked:

> I have a question. While i was reading the Blocking Users (Organizations) API, I find the following lines:
> ```
> The token used to authenticate the call must have the admin:org scope in order to make any
> blocking calls for an organization. Otherwise, the response returns HTTP 404.
> ```
> In authorizations.go we have all the scopes defines. I do not see any references to scope anywhere? Is there a central place where this scope validation is done?

Let me answer that here. That note is meant for users of GitHub API. When authenticating, they need to make sure the token they're using has that scope.

There's nothing we need to do for these endpoints inside go-github, that information is for users of go-github. Authentication is orthogonal matter, see https://github.com/google/go-github#authentication.

So, nothing needs to be done here, and you can safely remove these TODOs.

However, if you want, you can add that note to the documentation for these 4 endpoints. E.g.:

```
// ListBlockedUsers lists all the users blocked by an organization.
//
// The token used to authenticate the call must have the admin:org scope
// in order to make any blocking calls for an organization. Otherwise,
// the response returns HTTP 404.
//
// GitHub API docs: https://developer.github.com/v3/orgs/blocking/#list-blocked-users
```

We generally don't try to document this for all endpoints, people are expected to read GitHub API documentation to figure out what scopes are required. So I'm not sure if it's worth doing or not just here.